source/elements/oneDAL/source/common_interface/error_handling.rst

```diff
@@ -1,3 +1,120 @@
+.. _error_handling:
+
 ==============
 Error handling
 ==============
+
+oneDAL error handling relies on mechanism of C++ exceptions. If an error occurs,
+it must be propagated at the point of a function call where it can be caught
+using С++ exception handlers.
+
+Exceptions classification
+=========================
+
+Classification is aligned with C++ STL classification. oneDAL base exception
+classes are abstract. Leaf classes are also inherited from the corresponding STL
+exceptions. Only these classes can be thrown.
```

Comment:
```suggestion
Exception classification in onDAL is aligned with C++ STL classification. Base exception
classes in oneDAL are abstract. Leaf classes are also inherited from the corresponding STL
exceptions. Only these classes can be thrown.
```

`Only these classes can be thrown.` -- this sounds strange to me: you throw exceptions, not classes + maybe it should be merged with the previous sentence?

Reply:
also I'd spell-out the first usage of STL